go.mod

```diff
@@ -1,15 +1,18 @@
 module github.com/zclconf/go-cty
 
 require (
+	github.com/apparentlymart/go-cidr v1.0.1
 	github.com/apparentlymart/go-textseg v1.0.0
-	github.com/golang/protobuf v1.1.0 // indirect
+	github.com/bmatcuk/doublestar v1.1.5
```

Comment:
v1.2 of this pkg has a different behaviour; this causes the tests to fail for older versions of go (1.11 & 1.12) :

```
--- FAIL: TestFileSet (0.00s)
    --- FAIL: TestFileSet/FileSet(".",_cty.StringVal("."),_cty.StringVal("\\")) (0.00s)
        filesystem_test.go:297: succeeded; want error
```

Reply:
I'd like to fix this test and use v1.2 but I'm not sure if this will be a breaking change ?

Reply:
I'm not sure I fully understand the implications of this change in behavior right now. Could we leave this on v1.1.0 for the moment and consider the implications of upgrading this separately?

Terraform will be using Go 1.12 until its next major release, so in particular we need to be careful here not to do anything that would cause misbehavior under Go 1.12.

Reply:
Ah it turns out a go mod tidy removed it. it's in https://github.com/hashicorp/go-cty-funcs/pull/4 now !